python3/raygun4py/middleware/wsgi.py

```diff
@@ -1,48 +1,53 @@
 import logging
 
-from raygun4py import raygunprovider, http_utilities
-
+from raygun4py import http_utilities, raygunprovider
 
 log = logging.getLogger(__name__)
 
 
 class Provider(object):
 
-    def __init__(self, app, apiKey):
+    def __init__(self, app, apiKey, config=None):
         self.app = app
-        self.sender = raygunprovider.RaygunSender(apiKey)
+        self.config = config if config is not None else {}
```

Comment:
Does `config` need to be stored in this object? could it just be created locally and then passed into the `.RaygunSender(...)`? Same note for the flask middleware

Reply:
Resolved as this is needed for unit test